tf2_eigen/include/tf2_eigen/tf2_eigen.h

```diff
@@ -80,15 +89,20 @@ void fromMsg(const geometry_msgs::PointStamped& msg, tf2::Stamped<Eigen::Vector3
 template <>
 void doTransform(const tf2::Stamped<Eigen::Affine3d>& t_in,
 		 tf2::Stamped<Eigen::Affine3d>& t_out,
-		 const geometry_msgs::TransformStamped& transform) {
-  t_out = tf2::Stamped<Eigen::Affine3d>(transformToEigen(transform) * t_in, transform.header.stamp, transform.header.frame_id);
+		 const geometry_msgs::msg::TransformStamped& transform) {
+  std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds> converted{std::chrono::nanoseconds{transform.header.stamp.sec*1000000000LL+transform.header.stamp.nanosec}};
```

Comment:
Looking back at this, yes.  The problem is that all of the geometry_msgs have a header, and that header has a time.  The time is a builtin_interfaces::msg::Time_ object, and tf2::Stamped wants a std::chrono::time_point object.  I *could* also add a new constructor to tf2::Stamped that takes in a builtin_interfaces::msg::Time_ object instead; do you think that would be better?